jerry-core/ecma/operations/ecma-get-put-value.c

```diff
@@ -257,9 +257,19 @@ ecma_op_put_value_lex_env_base (ecma_object_t *lex_env_p, /**< lexical environme
 
             ecma_named_data_property_assign_value (lex_env_p, property_value_p, value);
           }
-          else if (is_strict)
+          else
```

Comment:
Do we need the extra indentation at all? Would it be possible to do something like this?
```c
#if ENABLED (JERRY_ES2015)
    else if (ecma_is_property_enumerable (*property_p))
    {
      // ...
    }
#endif /* ENABLED (JERRY_ES2015) */
    else if (is_strict)
    {
      // ....
    }
```